packages/components/src/components/accordion/accordion.jsx

```diff
@@ -33,7 +38,7 @@ const Accordion = ({ className, icon_close, icon_open, list }) => {
                     <div className='dc-accordion__item-header' onClick={() => onClick(idx)}>
                         {item.header}
                         <div className='dc-accordion__item-header-icon-wrapper'>
-                            {open_idx === idx ? (
+                            {expand_all || open_idx === idx ? (
```

Comment:
```suggestion
                            {is_expand_all || open_idx === idx ? (
```

Reply:
Will remove state and use the prop directly